include/ddc/kernels/splines/bsplines_uniform.hpp

```diff
@@ -372,8 +372,10 @@ class UniformBSplines : detail::UniformBSplinesBase
             return 1.0 / ddc::step<knot_discrete_dimension_type>();
         }
 
-        KOKKOS_INLINE_FUNCTION discrete_element_type
-        eval_basis(DSpan1D values, ddc::Coordinate<CDim> const& x, std::size_t degree) const;
+        KOKKOS_INLINE_FUNCTION discrete_element_type eval_basis(
+                DSpan1D values,
+                ddc::Coordinate<CDim> const& x,
+                [[maybe_unused]] std::size_t degree) const;
```

Comment:
I suspect it comes from the `[[maybe_unused]]` here but first I would like to see if the CI can display the error.